alacritty.yml

```diff
@@ -363,7 +363,8 @@ live_config_reload: true
 # Entries in `shell.args` are passed unmodified as arguments to the shell.
 #
 # Default:
-#   - (Linux/macOS) /bin/bash --login
+#   - (macOS) /bin/bash
+#   - (Linux) shell program defined in password file for current user
```

Comment:
I think this can be reduced to just `user login shell`, that should make things clear since it uses common terminology.